docker-compose.yml

```diff
@@ -86,5 +86,18 @@ services:
     networks:
       - laravel
 
+  phpunit:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: phpunit
+    user: 1000:1000
```

Comment:
Hi @GitHM may I know what `user: 1000:1000` does? Thanks

Reply:
@mrkalmdn it instructs the phpunit container to use userid/groupid 1000. this ensures read/write access with the other containers that use the shared volumes (bind mount './src')